src/jsx.d.ts

```diff
@@ -1725,6 +1730,7 @@ export namespace JSXInternal {
 		icon?: string | undefined | SignalLike<string | undefined>;
 		id?: string | undefined | SignalLike<string | undefined>;
 		indeterminate?: boolean | undefined | SignalLike<boolean>;
+		inert?: boolean | undefined | SignalLike<boolean>;
```

Comment:
```suggestion
		inert?: boolean | undefined | SignalLike<boolean | undefined>;
```

Missed the `undefined`, I think